metric/push/cache/count

```diff
@@ -0,0 +1,17 @@
+# 10 people, 45000 messages total
+# Operation, Count
```

Comment:
I was looking at the previous count result:
https://github.com/DIBL/mini_wechat/blob/master/metric/push/no_cache/count

Seems we did not clean up the database before running last time? There should be "db.users.insertOne" entries